README.md

```diff
@@ -22,3 +22,13 @@ The site is deployed at https://crds.stackable.tech with Netlify.
 
 The `static/halfmoon-variables.css` file is from the halfmoon UI framework, v1.1.1.
 There was a slight modification for navbar alignment.
+
+### How to add a new platform release
+
+The docs are built for all the repos and branches configured in the `repos.yaml` file.
+so if this file is updated and merged into main, the automated Netlify build will automatically build
```

Comment:
```suggestion
so if this file is updated and merged into main, the  Netlify GitHub action will automatically build
```

Reply:
It's not a Netlify GitHub action. Netlify watches the repo and does it all internally